python/tests/integration/test_retry.py

```diff
@@ -88,7 +88,7 @@ def test_retry_timeout():
                 headers={
                     're': RetryType.CONNECTION_ERROR_AND_TIMEOUT
                 },
-                ttl=0.005,
+                ttl=0.00001,
```

Comment:
The number is tweaked to decide whether the test gets a local Timeout based on TTL or receive an Protocol Error Frame from server.